src/pymovements/datasets/dataset.py

```diff
@@ -432,7 +432,9 @@ def pix2deg(self, verbose: bool = True) -> Dataset:
 
         return self
 
-    def pos2vel(self, method: str = 'smooth', verbose: bool = True, **kwargs) -> Dataset:
+    def pos2vel(
+            self, method: str = 'smooth', verbose: bool = True, **kwargs: int | float | str,
```

Comment:
If the function signature takes up more than a line I personally prefer to have each argument on a separate line.

Reply:
done